ui/src/store/api/public_keys.js

```diff
@@ -1,17 +1,17 @@
 import http from '@/store/helpers/http';
 
-export const postPublicKey = async (data) => http().post('/sshkeys/public_keys', {
+export const postPublicKey = async (data) => http().post('/sshkeys/public-keys', {
```

Comment:
@gustavosbarreto this part of the code, I switched to hyphens. But, to work it is necessary to make the exchange in the API. What do I do?